requirements.in

```diff
@@ -1,5 +1,6 @@
 autoflake==1.4
 black==21.12b0
+click==8.0.3
```

Comment:
it looks like we only installed this package but haven't started using it yet, or maybe I'm missing something?

Reply:
It's used directly by `__etl__/main.py` now (was previously only used by various dependencies)